cli/tsc/99_main_compiler.js

```diff
@@ -349,37 +360,27 @@ delete Object.prototype.__proto__;
           return undefined;
         }
       } else {
-        const sourceFile = sourceFileCache.get(specifier);
+        let sourceFile = sourceFileCache.get(specifier);
         if (sourceFile) {
           return sourceFile;
         }
 
-        try {
-          /** @type {{ data: string; hash: string; }} */
-          const { data, hash } = core.jsonOpSync(
-            "op_load_module",
-            { specifier },
-          );
-          const sourceFile = ts.createSourceFile(
-            specifier,
-            data,
-            languageVersion,
-          );
-          sourceFile.moduleName = specifier;
-          sourceFile.version = hash;
-          sourceFileCache.set(specifier, sourceFile);
-          return sourceFile;
-        } catch (err) {
-          const message = err instanceof Error
-            ? err.message
-            : JSON.stringify(err);
-          debug(`  !! error: ${message}`);
-          if (onError) {
-            onError(message);
-          } else {
-            throw err;
-          }
-        }
+        /** @type {{ data: string; hash: string; }} */
+        const { data, hash } = core.jsonOpSync(
+          "op_load",
+          { specifier },
+        );
+        debug({ data, hash });
```

Comment:
TIOLI - remove printing of source code before landing - I find it unhelpful when in debug log